src/util/lean_path.cpp

```diff
@@ -17,7 +17,8 @@ Author: Leonardo de Moura, Gabriel Ebner
 
 namespace lean {
 lean_file_not_found_exception::lean_file_not_found_exception(std::string const & fname):
-    exception(sstream() << "file '" << fname << "' not found in the LEAN_PATH"),
+    exception(sstream() << "file '" << fname << "' not found in the search path\n"
+        << "Use 'lean --path' to see where lean is looking, or https://leanproject-community.github.io/file-not-found for more"),
```

Comment:
```suggestion
        << "Use 'lean --path' to see where lean is looking, or https://leanproject-community.github.io/file-not-found.html for more"),
```
We could make it work without `.html` if we put effort into URL routing, but we haven't yet.

Reply:
```suggestion
        << "Use 'lean --path' to see where lean is looking, or https://leanprover-community.github.io/file-not-found.html for more"),
```
So who's going to write this page?

Reply:
Thanks, I missed that. 

I made a start on the file in [this PR](https://github.com/leanprover-community/leanprover-community.github.io/pull/23), but it definitely needs more material.

Reply:
@bryangingechen Thanks for taking care of the web site!